src/backend/metal/src/command.rs

```diff
@@ -774,7 +774,11 @@ impl CommandSink {
             CommandSink::Deferred { ref mut passes, ref mut is_encoding } => {
                 *is_encoding = keep_open;
                 passes.push(soft::Pass::Render {
-                    desc: descriptor.to_owned(),
+                    //Note: the original descriptor belongs to the framebuffer,
+                    // and will me mutated afterwards.
+                    desc: unsafe {
+                        msg_send![descriptor, copy]
```

Comment:
Could we somehow specify whether the `copy` is actually needed to avoid this when it hasn't changed?